gen/dcos-config.yaml

```diff
@@ -884,6 +884,18 @@ package:
     content: |
       MARATHON_ZK=zk://zk-1.zk:2181,zk-2.zk:2181,zk-3.zk:2181,zk-4.zk:2181,zk-5.zk:2181/marathon
       LIBPROCESS_PORT=15101
+      MARATHON_MESOS_MASTER_URL=zk://zk-1.zk:2181,zk-2.zk:2181,zk-3.zk:2181,zk-4.zk:2181,zk-5.zk:2181/mesos
```

Comment:
Are these variables going literally into `/etc_master/marathon`?

Reply:
yes

Reply:
How do the variables in this file get available to Marathon then?